docs/_data/table/en/user-configuration.yml

```diff
@@ -99,11 +99,12 @@ alluxio.user.short.circuit.enabled:
 alluxio.user.ufs.block.open.timeout.ms:
   N/A
 alluxio.user.ufs.block.read.concurrency.max:
-  N/A
+  The maximum concurrent readers for one UFS block on one Block Worker.
```

Comment:
actually, these docs are auto-generated now in branch master. 
The correct way should be updating `PropertyKey.java` with 
```
  public static final PropertyKey USER_UFS_BLOCK_OPEN_TIMEOUT_MS =
      new Builder(Name.USER_UFS_BLOCK_OPEN_TIMEOUT_MS)
          .setAlias(new String[]{"alluxio.user.ufs.block.open.timeout.ms"})
          .setDescription("The maximum concurrent readers for one UFS block on one Block Worker.")
          .setDefaultValue(300000)
          .build();
```